util/docker/maketest.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Runs tests and reports on them to github
+
+DIR=$(cd $(dirname $0) ; pwd -P)
+source ${DIR}/docker_common.sh
+
+# Get sha sum
+USER="jgkamat"
```

Comment:
Not sure if this username should be changed to georgeburdell, but it seems to be working now?